source/material_model/rheology/visco_plastic.cc

```diff
@@ -313,6 +346,7 @@ namespace aspect
                   // rescale the viscosity back to yield surface
                   if (current_stress >= yield_stress)
                     {
+                      // TODO: Appendix E omits the sqrt of the viscoelastic_strain_rate_invariant.
```

Comment:
This comment is not  helpful, because it is unclear what Appendix E refers to. Can you clarify the comment and why it is here?

Reply:
Sorry, that's a reminder that I need to discuss it with Bob. I'll remove it.